mne/viz/topomap.py

```diff
@@ -2382,3 +2382,158 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
     fig.canvas.draw()
     plt_show(show)
     return fig
+
+
+def _trigradient(x, y, z):
+    """Take gradients of z on a mesh."""
+    from matplotlib.tri import CubicTriInterpolator, Triangulation
+    tri = Triangulation(x, y)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        tci = CubicTriInterpolator(tri, z)
+    dx, dy = tci.gradient(tri.x, tri.y)
+    return dx, dy
+
+
+def plot_arrowmap(data, info_from, info_to=None, scale=1e-10, vmin=None,
+                  vmax=None, cmap=None, sensors=True, res=64, axes=None,
+                  names=None, show_names=False, mask=None, mask_params=None,
+                  outlines='head', contours=6, image_interp='bilinear',
+                  show=True, head_pos=None, onselect=None):
+    """Plot arrow map.
+
+    Parameters
+    ----------
+    data : array, shape (n_channels,)
+        The data values to plot.
+    info_from : instance of Info
+        The measurement info from data to interpolate from.
+    info_to : instance of Info | None
+        The measurement info to interpolate to.
```

Comment:
As a person who is not familiar with this method, the `info_from` and `info_to` parameters confuse me. Can you explain somewhere what data transformation is performed and why? Something that will guide me as to what to specify as `info_from` and `info_to` (the example seems to suggest this must always be grad versus mag data... is that correct?)

Reply:
Added one more example and add some description for `info_to`